webpack/assets/javascripts/react_app/components/notifications/index.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import * as NotificationActions from '../../redux/actions/notifications';
+import './notifications.scss';
+import ToggleIcon from './toggleIcon/';
+import Drawer from './drawer/';
+import { groupBy, some, isUndefined } from 'lodash';
+import { getNotificationsInterval } from './notifications.consts';
+
+class notificationContainer extends React.Component {
+  componentDidUpdate() {
+    window.tfm.tools.activateTooltips();
+  }
+
+  componentDidMount() {
+    const { getNotifications, data: { url } } = this.props;
+    const timerId = setInterval(
```

Comment:
IMHO SetInterval is not a good use here, as I would not want to fire API call every 10 seconds, but rather 10s from when the last response was.